phpBB/includes/mcp/mcp_post.php

```diff
@@ -416,6 +474,62 @@ function mcp_post_details($id, $mode, $action)
 }
 
 /**
+ * Get the number of posters for a given ip
+ *
+ * @param \phpbb\db\driver\driver_interface $db
+ * @param string $poster_ip
+ * @return int
+ */
+function phpbb_get_num_posters_for_ip(\phpbb\db\driver\driver_interface $db, $poster_ip)
```

Comment:
could try to deduplicate these methods, but without putting the column name directly into the query string, you loose the overview on them